tests/sles4sap/sapconf.pm

```diff
@@ -12,26 +12,29 @@ use testapi;
 use version_utils qw(is_staging is_sle is_upgrade);
 use Utils::Architectures;
 use Utils::Systemd 'systemctl';
+use utils qw(zypper_call);
 use strict;
 use warnings;
 
 sub run_developers_tests {
-    my $devel_repo = 'https://gitlab.suse.de/AngelaBriel/sapconf-test/repository/sapconf_v5/archive.tar.gz';
+    my $devel_repo = 'https://gitlab.suse.de/AngelaBriel/sapconf-test';
+    my $branch = 'sapconf_v5';
```

Comment:
It will be always same branch ? Could be variable with default value otherwise.

Reply:
> It will be always same branch ? Could be variable with default value otherwise.

Now that sapconf v5 dropped the tuned dependency, I think we'll stick with it for a long time.

Reply:
Sure, when it changes then it changes. :wink: